tests/backend/topology/conftest.py

```diff
@@ -18,8 +18,8 @@ def swarm():
         "position": np.array([[5, 5, 5], [3, 3, 3], [1, 1, 1]]),
         "velocity": np.array([[1, 1, 1], [1, 1, 1], [1, 1, 1]]),
         "current_cost": np.array([2, 2, 2]),
-        "pbest_cost": np.array([1, 2, 3]),
-        "pbest_pos": np.array([[1, 2, 3], [4, 5, 6], [7, 8, 9]]),
+        "pbest_cost": np.array([2, 3, 1]),
+        "pbest_pos": np.array([[4, 5, 6], [7, 8, 9], [1, 2, 3]]),
```

Comment:
So changing this still works as expected, right? Interesting. Glad you caught this bug. :+1: